src/styles/_catalog.scss

```diff
@@ -5,6 +5,8 @@
   max-width: 944px;
   margin: 0 auto;
 
+  perspective: 100px;
```

Comment:
but why?

Reply:
Agreed, there is no need of this property. 
Actually I was playing with this property and at the end I forget to remove this from the code.

Reply:
Agreed, there is no need of this property.
Actually I was playing with this property and at the end I forget to remove this from the code.